tasks/install_shell.sh

```diff
@@ -614,6 +614,7 @@ case $platform in
       "8") deb_codename="jessie";;
       "9") deb_codename="stretch";;
       "10") deb_codename="buster";;
+      "testing") deb_codename="bullseye";; # FIXME: to be fixed when Debian is released and major_version is changed to 11
```

Comment:
Unfortunately this won't work with the modules team setup as the matrix for it is automatically generated from the metadata file, with the resulting machine that is provisioned being clearly labelled as 11.

Reply:
Debian 11 is planned to be released middle of August and then `major_version` will be updated to `11` instead of `testing`.
Do you have any proposal on how to fix this in the meantime?

Reply:
I've tested and for our setup 11 seems to work fine.

Reply:
```
      "10") deb_codename="buster";;
      "11") deb_codename="bullseye";;
```
Adding the above to the code in my fixtures allowed me to install the agent on my docker machine and the acceptance tests ran fine.

Reply:
If testing is required for the setup that you are using, could we just include both? And then removed testing when it is no longer needed, it seems like the simplest option

Reply:
We can do `11|testing)` until after the official release. The docker images return "11" indeed, but our vmpooler boxes return "testing".

Reply:
I updated PR to support both for now